.github/workflows/pkgcheck.yml

```diff
@@ -24,12 +70,29 @@ jobs:
       if: runner.os == 'Linux' && matrix.packages
       run: |
         sudo apt-get update
-        sudo apt-get install -y ${{ matrix.packages }}
+        sudo apt-get install -y --no-install-recommends ${{ matrix.packages }}
```

Comment:
Consider adding `ninja-build` here instead of everytime above. Also under macOS we install ninja as part of the install packages step below by default.